ord_schema/frozen_message_test.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 The Open Reaction Database Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for ord_schema.frozen_storage."""
+
+import dataclasses
+
+from absl.testing import absltest
+
+from ord_schema import frozen_message
+from ord_schema.proto import reaction_pb2
+
+
+class FrozenMessageTest(absltest.TestCase):
+    def test_scalar_without_optional(self):
+        """This is what happens if scalar fields are not labeled `optional`."""
+        frozen = frozen_message.FrozenMessage(
+            reaction_pb2.StirringConditions.StirringRate())
+        self.assertTrue(hasattr(frozen, 'rpm'))  # Wrong!
```

Comment:
Confused by `# Wrong!` here. This seems like it's the expected (but undesired) behavior when we're not using the `optional` field modifier or proto2. Is this just a note so we know to revise this  test later?

Reply:
It's just a disclaimer that fields not marked `optional` will have inappropriate behavior. I thought it might be nice to illustrate the problem we are trying to solve.

Reply:
I can remove or expand the comment; what would you like me to do here?

Reply:
Expanding the comment would be helpful, it's good to have it in there